docs/usage/models.md

```diff
@@ -424,6 +424,21 @@ Where `Field` refers to the [field function](schema.md#field-customisation).
 Here `a`, `b` and `c` are all required. However, use of the ellipses in `b` will not work well
 with [mypy](mypy.md), and as of **v1.0** should be avoided in most cases.
 
+If you want to specify a field that can take a `None` value while still being required, you can use `Optional` with `...`:
+
+```py
+from typing import Optional
```

Comment:
please move this into a python file so it's linted and tested as with all other code examples.

Reply:
Sure. The example right above is also embedded in the document, should I move that too?

Reply:
Done.

Reply:
sorry about that, I guess it either got missed or I was lazy and thought it was short enough not to matter, I'll change it when I have a chance.